### ISSUE TITLE
Accept local coffee installs

### ISSUE THREAD

**Issue by binarykitchen** [aponxi/sublime-better-coffeescript#216]
Currently that plugin won't work with local coffee binaries, located inside `node_modules/bin` of your project.

In `CoffeeScript.sublime-settings` I added that line `"binDir": "node_modules/.bin",` yet, I get the error `FileNotFoundError: [Errno 2] No such file or directory: 'coffee'`. Do I really have to npm-install coffee globally?

**Comment by markalfred** [comment]
You could define an absolute path to the binary on a per-project basis, so you could have one default `binDir` in your `CoffeeScript.sublime-settings`, then a project-specific `binDir` in your `projectname.sublime-project` file. To define project-specific coffeescript settings, add them to a `"CoffeeScript"` section of the `"settings"` portion of your `.sublime-project` file.

For instance, for the project `sublime-better-coffeescript`, I'd have the following file:

*/users/markalfred/repos/sublime-better-coffeescript/project.sublime-project*
``` json
{
  "folders":
  [
    {
      "follow_symlinks": true,
      "path": "."
    }
  ],
  "settings":
  {
    "CoffeeScript":
    {
      "binDir": "/users/markalfred/repos/sublime-better-coffeescript/node_modules/.bin"
    }
  }
}
```

This should override your global `binDir` for this particular project. Let me know if this works for you

**Comment by binarykitchen** [comment]
yes, this worked

**Comment by binarykitchen** [comment]
no, this is still not good enough.  it should accept relative paths:

```
  "settings":
  {
    "CoffeeScript":
    {
      "binDir": "./node_modules/.bin"
    }
  }
```

because the .sublime-project file is shared among developers and everyone has a different absolute path. and `"./node_modules/.bin"` doesn't work.
